tests/libfixmath_unittests/Makefile

```diff
@@ -7,8 +7,6 @@ BOARD_BLACKLIST := arduino-mega2560 waspmote-pro arduino-uno arduino-duemilanove
 # The MSP boards don't feature round(), exp(), and log(), which are used in the unittests
 BOARD_BLACKLIST += chronos msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
 
-BOARD_INSUFFICIENT_MEMORY := nucleo-f031k6 nucleo-f042k6 nucleo-l031k6
```

Comment:
I don't think it is safe to remove this, according to my test above.

Reply:
Thank you for the review. It is cool that it fixes the upgrade.

I removed this line to get the report from murdock, I will re-re-re-revert when it fails.

Reply:
It indeed failed also on `murdock` I reverted the `BOARD_INSUFFICIENT_MEMORY` commit.